cmd/keeper/keeper.go

```diff
@@ -895,6 +895,12 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 						started = true
 					}
 				}
+
+				if err = p.resyncIfNotReady(followed, initialized, started); err != nil {
+					log.Error(err)
+					return
```

Comment:
From the code of [line 566](https://github.com/gravitational/stolon/blob/1305863d645d7c69beaf16495152ec0001e2d7f6/cmd/keeper/keeper.go#L566) it will try this whole function again after sleep interval. I am also not fully familiar with this code, just made the same as in a lot of different places here.